src/Facade/Services/PurchaseOrderDeliveryFacadeService.cs

```diff
@@ -86,10 +86,10 @@ public class PurchaseOrderDeliveryFacadeService : IPurchaseOrderDeliveryFacadeSe
 
                     var firstFormatSatisfied =
                         DateTime.TryParseExact(row[2]
-                            .Trim(), "dd/MM/yyyy", CultureInfo.InvariantCulture, DateTimeStyles.None, out var parsedDate);
+                            .Trim(), "dd'/'M'/'yyyy", CultureInfo.InvariantCulture, DateTimeStyles.None, out var parsedDate1);
```

Comment:
Do you need the single quotes inside? Also what's one M, full month?

Reply:
I'm not sure really. It didn't work so i google it and this suggestion worked